File: 27-RemoveElement/27-RemoveElement.java
// Last updated: 8/1/2025, 1:58:29 PM
class Solution {
    public int removeElement(int[] nums, int val) {
        int j=0;
		for(int i=0;i<nums.length;i++) {
			if(nums[i]!=val) {
				nums[j]=nums[i];
                j++;
			}
		}
		return j;
    }
}

File: 27-RemoveElement/27-RemoveElement.py
# Last updated: 1/22/2026, 8:54:41 PM
1class Solution:
2    def removeElement(self, nums: List[int], val: int) -> int:
3        j=0
4        for i in range (len (nums)):
5            if nums[i]!=val:
6                nums[j]=nums[i]
7                j+=1
8
9        return j
10        